tutorials/katacoda/thanos/2-lts/step3.md

```diff
@@ -0,0 +1,74 @@
+In this step, we will configure object store and change sidecar to upload to object store.
+
+## Configuration :
+
+Click snippet to create a new file named "bucket_storage.yml"
+
+The configuration file content :
+
+```
+touch bucket_storage.yml
+```{{execute}}
+
+Click `Copy To Editor` for config to propagate the configs to the file `bucket_storage.yml`:
+
+<pre class="file" data-filename="bucket_storage.yml" data-target="replace">
+type: S3
+config:
+  bucket: "thanos"
+  endpoint: "minio:9000"
+  insecure: true
+  signature_version2: true
+  access_key: "minio"
+  secret_key: "minio123"
+</pre>
+
+Now, execute the following command :
+
+```
+docker run -d --net=host --rm \
+    --name sidecar \
+    quay.io/thanos/thanos:v0.12.2 \
+    sidecar \
+    --tsdb.path                 /prometheus \
+    --objstore.config-file      bucket_storage.yml \
+    --prometheus.url            http://127.0.0.1:9090 \
+    --http-address              0.0.0.0:19190    \
+    --grpc-address              0.0.0.0:19090
+```{{execute}}
+
+The flag `--objstore.config-file` loads all the required configuration from the file in order to ship the TSDB blocks to an object storage bucket, the storage end points and the credentials used.
+
+## Running Minio
+
+Click `Copy To Editor` for each to propagate the configs to the file.
```

Comment:
Propagate changes to what? (: what we want to achieve in this step. Let's be explicit, focused and clear what we want to achieve. And what user with get from this (: